src/Form/Ingest/Review.php

```diff
@@ -146,13 +146,19 @@ public function submitActivation(array &$form, FormStateInterface $form_state) {
    *   A reference to the batch context.
    */
   public function runBatchOp(MigrationInterface $migration, MigrateExecutable $e, &$context) {
+    assert($e instanceof MigrationRollbackBatch);
```

Comment:
```suggestion
    assert($e instanceof MigrationRollbackBatch || $e instanceof MigrateBatchExecutable);
```

Reply:
Tested this. With this change, I was able to run both ingest and rollback in the UI. Committing change.